plugin/src/main/java/com/playmonumenta/epicstructures/managers/RespawningStructure.java

```diff
@@ -374,6 +453,17 @@ public void tick(int ticks) {
 
 		mTicksLeft -= ticks;
 
+		boolean empty = true;
+		for (Player player : Bukkit.getOnlinePlayers()) {
+			if (mInnerBounds.within(player.getLocation().toVector())) {
+				empty = false;
+			}
+		}
```

Comment:
We definitely don't want to do this here. This will check all 30 players online for every 100+ R2 POIs. (> 3000 potential bounding box checks every few ticks).